openerp/addons/base/migrations/8.0.1.3/pre-migration.py

```diff
@@ -47,3 +47,12 @@ def migrate(cr, version):
         cr, 'ir_ui_view', 'type', [
             'calendar', 'diagram', 'form', 'gantt', 'graph', 'kanban',
             'qweb', 'search', 'tree'])
+    
+    #The tables stock.picking.in and stock.picking.out are merged into stock.picking
```

Comment:
This comment should be have a space between the `#` and the first word and also check 80 cols limit.